### PR TITLE
Use requirements that exist in bioconda

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: tool_collections/taxonomy/kraken2tax/kraken2tax.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <tool id="Kraken2Tax" name="Convert Kraken" version="1.1">
```
Comment (nsoranzo) [inline tool_collections/taxonomy/kraken2tax/kraken2tax.xml]: Can you increase the tool version?

**Comment by nsoranzo** [comment]
We can also remove `tool_dependencies.xml` at this point, I think.

**Comment by davebx** [comment]
Freedom eyes strikes again.

**Comment by bgruening** [comment]
Thanks @davebx and @nsoranzo!

**Comment by bgruening** [comment]
@davebx more of these!!!

**Comment by nsoranzo** [comment]
I just noticed that this tool is not properly tested because `tool_collections/taxonomy/kraken2tax` is still in `.tt_blacklist` . Can you open an additional PR which removes this line and also does a small change to the tool to trigger the planemo tests?

**Pull request merged**